ykrt/src/compile/jitc_yk/codegen/x64/lsregalloc.rs

```diff
@@ -1143,6 +1149,15 @@ impl LSRegAlloc<'_> {
                 }
                 _ => todo!("{bits}"),
             },
+            SpillState::ConstPtr(v) => {
+                // unwrap cannot fail since pointers are sized.
+                let bitw = self.m.type_(self.m.ptr_tyidx()).bitw().unwrap();
+
+                match bitw {
```

Comment:
This is an `assert` masquerading as a `todo` I think?

Reply:
Yeah, just the result of copying code from elsewhere :) Will fix.

Reply:
fixed in f9da2df36725f8e28714bb64acd9e52604b9c73f